packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -266,7 +265,7 @@ export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
         };
 
         const processedSeries = (options.series as SeriesOptionsTypes[])?.map((series) => {
-            series.type ??= defaultSeriesType;
+            series.type ??= this.getDefaultSeriesType(options);
```

Comment:
NOTE: This now lazily calls `this.getDefaultSeriesType()` if there is no `series.type`.